libc/src/__support/time/windows/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+add_object_library(
+  clock_gettime
+  HDRS
+    ../clock_gettime.h
+  SRCS
+    clock_gettime.cpp
+  DEPENDS
+    libc.hdr.types.struct_timespec
+    libc.hdr.types.clockid_t
+    libc.hdr.errno_macros
+    libc.include.llvm-libc-macros.windows.time_macros_ext
```

Comment:
it doesn't seem like this direct dependency is used anymore.

Reply:
nice catch!